python/app/console.py

```diff
@@ -236,19 +238,20 @@ def open(self, path=None):
             return
 
         # clear the contents, open and load the file
-        fh = open(path)
-        try:
+        with io.open(path, "r", encoding="utf-8") as fh:
```

Comment:
It will be interesting to see if this becomes a pattern for us in the future, having to use `io.open`.

Reply:
Could you leave a comment in the code explaining why you need to do this instead of simply calling `open`. Maybe also leave a FIXME comment saying it makes things better, but not quite perfect for unicode strings.

Reply:
Sounds good!